### PR TITLE
Update README for Ehcache 3.5

### PR DESCRIPTION
**Pull request by cljohnso**
Updates for 3.5 and minor corrections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,32 +101,28 @@ Once your pull request has been merged, the site is deployed automatically.
 1. Ensure you first edit `_data/authors.yml` to add you information
```
Comment (henri-tremblay) [inline README.md]: "you" should be "your"

File: README.md [hunk 2]
```diff
@@ -99,32 +101,28 @@ Once your pull request has been merged, the site is deployed automatically.
...
-categories: blog
-hidden: false
-author: your_author_key
-tags: [ehcache, website]
-date: 2016-05-04T18:00:00-05:00
-modified:
----
-```
+   ```
+   layout: post
```
Comment (henri-tremblay) [inline README.md]: The `---` are normal. They are the front matter delimiters.

File: README.md [hunk 3]
```diff
@@ -99,32 +101,28 @@ Once your pull request has been merged, the site is deployed automatically.
...
+   date: 2016-05-04T18:00:00-05:00
+   modified:
+   ```
 4. Send your pull request :+1:
 
 ### Contributing an external resource
 
 Same rules apply as above, except that you have to put your file into `_posts/resources` and add a header like this one:
 
 ```
```
Comment (henri-tremblay) [inline README.md]: Same here. Delimiters should be kept

File: README.md (hunk 2 above)
Comment (cljohnso) [reply README.md]: K ... I'll put those back.  The indentation will remain since that's the proper way using MarkDown to continue content.

Comment (henri-tremblay) [reply README.md]: Sure.

**Pull request merged**